app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Admin
+  class SessionsController < ApplicationController
+    def new
+    end
+    def create
+      if params[:email].present?
+        email = params[:email].downcase
+      else
+        email = params[:email]
+      end
+      admin_user = User.find_by(email: email)
```

Comment:
to powinna być oddzielna metoda prywatna

Reply:
zrobione